config/api/models/System.php

```diff
@@ -52,12 +52,8 @@
         'requirements' => function (System $system) {
             return $system->toArray();
         },
-        'site' => function () {
-            try {
-                return $this->site()->blueprint()->title();
-            } catch (Throwable $e) {
-                return $this->site()->title()->value();
-            }
+        'site' => function (System $system) {
+            return $system->title();
```

Comment:
I think it's weird that we now have a `site` field that returns the content title with blueprint fallback and a `title` field that only returns the content title. Can we unify this somehow?

To be honest I don't fully understand what the `site` and `title` are used for respectively.

Reply:
I think we used it in different ways in the Panel. I don't think we use these anymore with Fiber,  but I also didn't want to remove them from the API as another breaking change.

Reply:
IMO we should check this and deprecate one of them, ideally `site`. Keeping both around is really confusing.

Reply:
As far as I can tell, on the JS we only use `$api.system.install()` from the system API  - `get()` and `register()` are completely unused. So from the Panel's side, we can do pretty much what we want in the API System model.

Reply:
We might even move the `install()` call to Fiber as well and consider deprecating the whole System part of the API.

But maybe we move that to its own issue and leave it here as is.

Reply:
Your call, it's only something I noticed and where I feel like we shouldn't leave it like that in the long term.

Reply:
https://github.com/getkirby/kirby/issues/3541